### PR TITLE
add tast solution

### PR DESCRIPTION
**Pull request by AnhelinaSh2001**
- [DEMO LINK](https://AnhelinaSh2001.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://AnhelinaSh2001.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -17,6 +21,86 @@
...
+            class="logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav__container">
+        <ul>
+          <li>
+            <a
+              class="nav__link"
```
Comment (anastasiiavorobiova) [inline src/index.html:39]:
```suggestion
              class="nav__link is-active"
```

File: src/index.html [hunk 2]
```diff
@@ -17,6 +21,86 @@
...
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link is-active"
```
Comment (anastasiiavorobiova) [inline src/index.html:63]:
```suggestion
              class="nav__link"
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,85 @@
...
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.logo-container {
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  top: 10px;
+  left: 50px;
```
Comment (anastasiiavorobiova) [inline src/style.css:19]:
```suggestion

```
Consider leveraging document flow

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,85 @@
...
+  color: rgba(0, 0, 0, 1);
+  display: flex;
+  align-items: center;
+  padding: 0;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__link:hover {
+  color: rgba(0, 172, 220, 1);
```
Comment (anastasiiavorobiova) [inline src/style.css:39]: Consider using variables for repeating colors

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,85 @@
...
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: rgba(0, 172, 220, 1);
+}
+
+.nav__container {
+  align-items: center;
+  padding-left: 419px;
```
Comment (anastasiiavorobiova) [inline src/style.css:55]:
```suggestion

```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,85 @@
...
+.nav__container li {
+  display: inline-block;
+  margin-right: 20px;
+}
+
+.nav__container li:last-child {
+  margin-right: 0;
+}
+
+.nav__container a {
```
Comment (anastasiiavorobiova) [inline src/style.css:69]: Consider using classes for styling

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,85 @@
...
+  height: 23px;
```
Comment (anastasiiavorobiova) [inline src/style.css:70]: The height should be 60px

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added comments to fix

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Well done, pls try to pass all tests from backstop before asking for review again)
Pls ask for help in **fe_chat** if you don't understand something

**Pull request closed without merging**